db/org_participants.go

```diff
@@ -207,3 +208,83 @@ func (ms *MongoStorage) BulkUpsertOrgParticipants(
 
 	return result, nil
 }
+
+// OrgParticipants retrieves a orgParticipants from the DB based on it ID
+func (ms *MongoStorage) OrgParticipants(orgAddress string) ([]OrgParticipant, error) {
+	if len(orgAddress) == 0 {
+		return nil, ErrInvalidData
+	}
+	ms.keysLock.Lock()
+	defer ms.keysLock.Unlock()
+	// create a context with a timeout
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	cursor, err := ms.orgParticipants.Find(ctx, bson.M{"orgAddress": orgAddress})
+	if err != nil {
+		return nil, fmt.Errorf("failed to get orgParticipants: %w", err)
+	}
+	defer func() {
+		if err := cursor.Close(ctx); err != nil {
+			log.Warnw("error closing cursor", "error", err)
+		}
+	}()
+
+	var orgParticipants []OrgParticipant
+	if err = cursor.All(ctx, &orgParticipants); err != nil {
+		return nil, fmt.Errorf("failed to get orgParticipants: %w", err)
+	}
+
+	return orgParticipants, nil
+}
```

Comment:
I have almost the same method in my PR but paginated, so we must check how both can be merged to reuse code if is possible.